sections/futures/MarketDetails/MarketDetails.tsx

```diff
@@ -35,22 +37,45 @@ type MarketDetailsProps = {
 	mobile?: boolean;
 };
 
+interface OpenInterestDetailProps extends MarketDetailsProps {
+	isLong: boolean;
+}
+
 const MarketDetails: React.FC<MarketDetailsProps> = ({ mobile }) => {
 	const dispatch = useAppDispatch();
 	const showHistory = useAppSelector(selectShowHistory);
+
+	const MarketDetailsDesktop = () => (
```

Comment:
Would be good to render these conditionally, as currently a waste rendering both but only using one currently